src/hooks/api/users/useRemoveUserAddress.ts

```diff
@@ -37,7 +37,7 @@ export default function useRemoveUserAddress() {
       await mutate(
         getUserCacheKey({ id: user.id }),
         (user: User) => {
-          const addresses = user.addresses.filter((address) => address !== addressToRemove);
+          const addresses = user?.addresses.filter((address) => address !== addressToRemove);
```

Comment:
small fix for a bug I saw when removing addresses from my user where user was undefined.
it otherwise didn't seem to affect behavior and we removing swr and these mutations soon so I don't think further investigation is necessary